compose/mpp/demo/src/uikitMain/kotlin/bugs/RedundantRecompositionOnStart.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bugs
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Text
+import androidx.compose.mpp.demo.Screen
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.LocalDensity
+
+private var counter = 0
+
+// https://github.com/JetBrains/compose-multiplatform/issues/3778
+val StartRecompositionCheck = Screen.Example("Start Recomposition Check") {
+    remember {
+        counter = 0
+    }
```

Comment:
I guess it won't show a repro for fixed case. it calls `remember`'s lambda twice and assign zero to it. I guess this block can be removed

Reply:
Actually, this demo doesn't reproduce the issue at all, because we can only start it manually by navigating in the demos menu. By the moment it starts, the LocalDensity is already properly set and no extra recomposition occurr in this demo. 

I initially tested it on a top level and then moved the demo in iOS bugs category. We need something else to test this.